server/moodle/mod/livequiz/tests/phpunit/livequiz_service_test.php

```diff
@@ -0,0 +1,193 @@
+<?php
+// This file is part of Moodle - http://moodle.org/
+//
+// Moodle is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+//
+// Moodle is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+//
+// You should have received a copy of the GNU General Public License
+// along with Moodle.  If not, see <http://www.gnu.org/licenses/>.
+
+/**
+ * LiveQuiz Service Test Class
+ *
+ * This class contains unit tests for the LiveQuiz service functionality.
+ *
+ * @package mod_livequiz
+ * @copyright 2023
+ * @license http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
```

Comment:
where did this license come from?

Reply:
Same license used in other files. Is it not the right one?

https://github.com/AAU-P5-Moodle/moodle-1/blob/main/server/moodle/mod/livequiz/view.php
view.php for instance:

![image](https://github.com/user-attachments/assets/e2b0027e-d723-4cfe-a098-216cd3a4b676)

Do you want it updated to:

```
 * LiveQuiz Service Test Class
 *
 * This class contains unit tests for the LiveQuiz service functionality.
 *
 * @package   mod_livequiz
 * @copyright 2024 Software AAU
 * @license   http://www.gnu.org/copyleft/gpl.html GNU GPL v3 or later
 */
```